module/muxtweakgen.c

```diff
@@ -500,6 +587,7 @@ void init_elements() {
     lv_obj_set_user_data(ui_lblHidden, "hidden");
     lv_obj_set_user_data(ui_lblBGM, "bgm");
     lv_obj_set_user_data(ui_lblSound, "sound");
+    lv_obj_set_user_data(ui_lblAudioSink, "audio sink"); //hmm what does this even do
```

Comment:
This is used by glyphs and also used as a reference point within the frontend. This should be a single word, "audiosink" will do just fine without the space.